tests/unit_test/civicrm_email/civicrm_email_report_test.py

```diff
@@ -0,0 +1,79 @@
+from data_pipeline.civicrm_email_report.civicrm_email_report import (
+    transform_email_report
+)
+
+MAIL_ID = 100
+
+DEFAULT_CIVICRM_EMAIL_REPORT_RESPONSE = {
+    "values": {
+        str(MAIL_ID): {
```

Comment:
Just curious, the mail id is a string in the response, but an integer in other responses?

Reply:
when it is a key in dictionary it is a string. But when we sending it to API we are sending as an integer.

Reply:
Interesting. Although thinking about it more, it makes sense. As integers aren't valid JSON keys.